runtime/handler.go

```diff
@@ -61,7 +61,18 @@ func ForwardResponseStream(ctx context.Context, mux *ServeMux, marshaler Marshal
 			return
 		}
 
-		buf, err := marshaler.Marshal(streamChunk(ctx, resp, mux.streamErrorHandler))
+		var buf []byte
+		switch {
+		case resp == nil:
+			buf, err = marshaler.Marshal(errorChunk(streamError(ctx, mux.streamErrorHandler, errEmptyResponse)))
+		default:
+			if rb, ok := resp.(responseBody); ok {
+				buf, err = marshaler.Marshal(rb.XXX_ResponseBody())
```

Comment:
Please wrap this with the result envelope
```suggestion
				buf, err = marshaler.Marshal(map[string]proto.Message{"result": rb.XXX_ResponseBody()})
```

Reply:
May i know reason to wrap this with result envelope ?

Reply:
Of course :). The result envelope is there so that we can return a consistent structure for streaming messages, which can fail at any point in the stream. The envelope always has the structure `{"result":<message>, "error": <any error}`. This guarantees consistent JSON parsing. I want to preserve this property for this feature. If you need complete control over the streaming output, you may want to consider the `google.protobuf.HttpBody` type which is printed verbatim to the output.

Reply:
`rb.XXX_ResponseBody()` is not of proto.Message type. it will be `marshaler.Marshal(map[string]interface{}{"result": rb.XXX_ResponseBody()})`. is it ok ?

Reply:
What type is it? Can we make it anything other than `interface{}`? If it'

Reply:
Hm, I see, it's `interface{}`. Yeah I guess that will have to do :)